common/scripts/base.sh

```diff
@@ -139,3 +138,25 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
+{
+  if ! bash ${mesos_artifacts_home}/common/${1}/deploy.sh; then
+    echoError "Non-zero exit code returned when deploying ${1}"
+    exit 1
+  fi
+}
+
+function deploy_wso2_service()
```

Comment:
changed it to 'deploy_service' in commit adc6a6b8d30f2878b8754184726842cd7ebc7b07.